libsolidity/formal/CHC.h

```diff
@@ -275,12 +270,24 @@ class CHC: public SMTEncoder
 	//@{
 	struct CHCVerificationTarget: VerificationTarget
 	{
-		smtutil::Expression errorId;
+		unsigned errorId;
+		ASTNode const * const errorNode;
+	};
+
+	struct CHCVerificationTargetPlaceholder
+	{
+		smtutil::Expression constraints;
+		smtutil::Expression errorExpression;
+		smtutil::Expression startingBlock;
 	};
 
 	/// Verification targets corresponding to ASTNodes. There can be multiple targets for a single ASTNode,
 	/// e.g., divByZero and Overflow for signed division.
-	std::map<ASTNode const*, std::vector<CHCVerificationTarget>, IdCompare> m_verificationTargets;
+	std::map<ASTNode const*, std::vector<unsigned>, IdCompare> m_functionVerificationPreTargetIds;
```

Comment:
Ok so every time an error condition is seen, a new error id is created and added to the current function or contract in this variable, right?

Reply:
Please update the documentation of the variable.

Reply:
That's correct. Documentation updated.